README.md

```diff
@@ -40,6 +40,10 @@ The workflow contains the following documents:
 
```

Comment:
Please remove .DS_Store files from the PR

Reply:
@ritesh-modi there was a `.DS_Store` commited to the repo in the past. This PR has removed it. I guess that should be okay.